### ISSUE TITLE
Add docker workflow

### ISSUE THREAD

**Issue by singsuyash** [singsuyash/enterprise-platform#2]
On build, the generated docker image should be pushed to github packages
the docker images should be versioned, and the latest version should be tagged to latest.

Out of scope:
- docker image test

**Comment by ashwinimanoj** [comment]
Do we have a plan for versioning?

**Comment by ashwinimanoj** [comment]
We could also think about [changelog](https://keepachangelog.com/en/1.0.0/).

**Comment by singsuyash** [comment]
yeah, i am not sure how to version artefacts, but let me add this to the scope.
can changelogs be generated automatically by the git commit history?
